src/controllers/gameController.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+
+const gameServes: {
+  [id: string]: (req: express.Request, res: express.Response) => void;
+} = {
+  pacman(req: express.Request, res: express.Response) {
+    res.redirect('/static/pacman/index.html');
+  },
+};
+
+export const gameController = {
+  getGame(req: express.Request, res: express.Response) {
+    const internalGameName: string = req.params['game_name'];
+    const game = gameServes[internalGameName];
+    game(req, res);
+  },
+};
```

Comment:
Is this still the best way to handle things? I just feel like the pattern of finding a routing function from an object and then calling that function with req and res only to send a redirect response is just a bit confusing. Would something like this with [middleware](https://expressjs.com/en/guide/using-middleware.html) for the patched files and static serving for the rest work?
```node
/* src.controllers/gameController.ts */
export const gameController = {
  patchingMiddleware(req, res) {
    if(req.originalUrl === '/games/pacman/route/to/file/I/want/to/patch.js') {
      res.send('patched_file.js');
    }
    // continue patching for all cases of req.originalUrl you want to patch
   
    next();  // use static serving for non-patched files
  }
}
```
```node
/* src/routes/games.ts */
router.use('/', gameController.patchingMiddleware);
router.use('/', express.static('vendor'));
```

Reply:
Done in df2b5b1